mllib/src/main/scala/org/apache/spark/ml/feature/CountVectorizer.scala

```diff
@@ -127,6 +146,9 @@ class CountVectorizer(override val uid: String)
   /** @group setParam */
   def setMinTF(value: Double): this.type = set(minTF, value)
 
+  /** @group setParam */
+  def setBinary(value: Boolean): this.type = set(binary, value)
+
   setDefault(vocabSize -> (1 << 18), minDF -> 1)
```

Comment:
Yes agree with that

Sent from my iPhone

> On 6 Apr 2016, at 19:00, Bryan Cutler <notifications@github.com> wrote:
> 
> In mllib/src/main/scala/org/apache/spark/ml/feature/CountVectorizer.scala:
> 
> > @@ -127,6 +146,9 @@ class CountVectorizer(override val uid: String)
> >    /** @group setParam */
> >    def setMinTF(value: Double): this.type = set(minTF, value)
> >  
> > +  /** @group setParam */
> > +  def setBinary(value: Boolean): this.type = set(binary, value)
> > +
> >    setDefault(vocabSize -> (1 << 18), minDF -> 1)
> I think you should set the binary param default here. It does get called in the trait, but imo a trait should not really be doing things like that and a default value is better set by the class using it.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>